src/fstar/FStar.PushHelper.fs

```diff
@@ -0,0 +1,367 @@
+(* FStar.Interactive.Lsp and FStar.Interactive.Ide need to push various *
```

Comment:
Should its name be `FStar.Interactive.PushHelper.fs`, to say that it is used only by the interactive mode(s)? And how about `JsonHelper`, is that also interactive mode specific, or can it also be used by any other code that manipulates JSON?

Reply:
JsonHelper can be used by any code that manipulates JSON, but PH and QH need the `Interactive` prefix. Thanks for catching this.